src/app/downloads/downloads.component.html

```diff
@@ -66,7 +66,7 @@ <h2 class="card-title mb-2"><fa-icon [icon]="['fab', 'github']"></fa-icon> Old R
       <p class="mb-2 col-sm-12">Old releases of EnergyPlus (version 7.0.0 through 8.1.0) can be downloaded from the EnergyPlusRelease GitHub Repository.</p>
       <a class="btn btn-primary btn-bg-dark ml-3" href="https://github.com/NREL/EnergyPlusRelease/releases" role="button">View Old Releases</a>
       <p class="mt-3 col-sm-12">Transition utilities are available for updating older EnergyPlus input files to later versions. The files can be downloaded from the Helpdesk.</p>
-      <a class="btn btn-primary btn-bg-dark ml-3" href="http://energyplus.helpserve.com/Knowledgebase/List/Index/46/converting-older-version-files" role="button">Older Transition Programs</a>
+      <a class="btn btn-primary btn-bg-dark ml-3" href="https://energyplushelp.freshdesk.com/support/solutions/folders/67000579810" role="button">Older Transition Programs</a>
```

Comment:
I spent a little time with the wayback machine and playing detective to try to find reasonable new URLs.  In most cases, I was successful, others were more difficult.  But I think it's way better now than just having broken links.